<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClangTidy::PerProjectConfig</class>
 <widget class="QWidget" name="ClangTidy::PerProjectConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>379</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Header filter regex:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="headerFilterText">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Regular expression matching the names of the headers to output diagnostics from. Diagnostics from the main file of each translation unit are always displayed. Can be used together with -line-filter. This option overrides the value read from a .clang-tidy file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>E&amp;xtra parameters:</string>
       </property>
       <property name="buddy">
        <cstring>clangtidyParameters</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="clangtidyParameters">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Additional command line options to pass to clang-tidy.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="sysHeadersCheckBox">
       <property name="text">
        <string>Check system headers</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="overrideConfigFileCheckBox">
       <property name="text">
        <string>Override .clang-tidy file options</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="checkListGroupBox">
     <property name="toolTip">
      <string>Enable additional checks</string>
     </property>
     <property name="title">
      <string>Available Checks</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="KFilterProxySearchLine" name="checkFilterEdit"/>
      </item>
      <item>
       <widget class="QListView" name="checkListView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dumpCheckBox">
        <property name="text">
         <string>Dump active configuration to .clang-tidy file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFilterProxySearchLine</class>
   <extends>QWidget</extends>
   <header>kfilterproxysearchline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>overrideConfigFileCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkListGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
